engine/src/test/resources/log4j.properties

```diff
@@ -17,7 +17,7 @@
 # under the License.
 #
 
-log4j.rootLogger=INFO,CONSOLE
+log4j.rootLogger=INFO,CONSOLE,malhar
```

Comment:
The change at minimum impacts time to run unit tests and affects all unit test and all contributors 👎